lms/templates/admin/macros.jinja

```diff
@@ -63,57 +63,51 @@
     {{ form_text_field(request, "Token URL", "token_url", lti_registration.token_url) }}
 {% endmacro %}
 
-{% macro instances_table(request, instances) %}
-<table class="table is-fullwidth">
-  <thead>
-    <tr>
-      <th>ID</th>
-      <th>Consumer Key</th>
-      <th>GUID</th>
-      <th>Issuer</th>
-      <th>Deployment ID</th>
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
-{% for instance in instances %}
-    <tr>
-      <td>{{instance.id}}</td>
-      <td>{{instance.consumer_key}}</th>
-      <td>{{instance.tool_consumer_instance_guid}}</td>
-      <td>{{instance.lti_registration.issuer}}</td>
-      <td>{{instance.deployment_id}}</td>
-      <td>
-        <a class="button" href="{{ request.route_url("admin.instance.id", id_=instance.id) }}">View</a>
-      </td>
-    </tr>
-{% endfor %}
+
+{% macro created_updated_fields(model) %}
+    {{ disabled_text_field("Created", model.created) }}
+    {% if model.updated %}
+        {{ disabled_text_field("Updated", model.updated) }}
+    {% endif %}
+{% endmacro %}
+
+{% macro object_list_table(request, route, objects, headers, fields) %}
+<div class="container">
+    <div class="table-container">
+        <table class="table is-fullwidth">
+          <thead>
+            <tr>
+              <th></th>
+              {% for heading in headers %}
+              <th>{{heading}}</th>
+              {% endfor %}
+            </tr>
+          </thead>
+          <tbody>
+            {% for object in objects %}
+                <tr>
+                  <td>
+                    <a class="button" href="{{ request.route_url(route, id_=object.id) }}">View</a>
+                  </td>
+                  {% for field in fields %}
+                  <td>{{object[field]}}</th>
+                  {% endfor %}
+               </tr>
+            {% endfor %}
+          </tbody>
```

Comment:
`tbody` was not closed

Reply:
The view button on the left is a bit out of step with all the other buttons we have. I can see that when making the page narrower it means the button can still be reached.

I don't think this is quite right yet, as making page narrower has horizontal scroll, but only up to the min width of the container, which doesn't collapse.

So we have full width:

![image](https://user-images.githubusercontent.com/7131143/189946011-e304ec0a-8dfd-417b-87fd-7954e36deaab.png)

Within the stretch of the container:

![image](https://user-images.githubusercontent.com/7131143/189946155-dcfb44bd-c416-4184-b7ce-6fb1577cfffa.png)

But beyond this it doesn't get any smaller:

![image](https://user-images.githubusercontent.com/7131143/189946273-9acde24e-f92f-4840-b71d-37f0cb851c88.png)

I don't think we need to fix this in this PR, but we might want to log something about it.